README.md

```diff
@@ -7,17 +7,13 @@ The Ansible Juniper Networks Junos collection includes a variety of Ansible cont
 This collection has been tested against Juniper Networks Junos OS 18.4R1.
 
 <!--start requires_ansible-->
-
 ## Ansible version compatibility
 
 This collection has been tested against following Ansible versions: **>=2.9.10**.
 
-For collections that support Ansible 2.9, please ensure you update your `network_os` to use the
```

Comment:
This line is needed. Not sure why this line got removed. Can you please add this back?